src/components/Survey/Survey.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { Redirect } from 'react-router-dom';
+
+import { Form, Box, Heading } from 'grommet';
+
+import { saveSurvey } from '../../services/api';
+import SurveyField from '../SurveyField/SurveyField';
+import SurveyButton from '../Button';
+
+const messages = {
+  required: 'This field is required'
+};
+
+class Survey extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
Why don't u use hooks, the same as in main component ;)? Just stick with the one method.